kythe/go/storage/tools/triples/triples.go

```diff
@@ -156,6 +158,9 @@ func toTriple(entry *spb.Entry) (*rdf.Triple, error) {
 	if graphstore.IsEdge(entry) {
 		t.Predicate = entry.EdgeKind
 		t.Object = kytheuri.FromVName(entry.Target).String()
+	} else if entry.FactName == facts.Code {
```

Comment:
Test?

Reply:
There isn't one, and it's not worth the trouble to add one for this.